sentry-rails/spec/sentry/rails/tracing/active_record_subscriber_spec.rb

```diff
@@ -32,6 +32,29 @@
       expect(span[:description]).to eq("SELECT \"posts\".* FROM \"posts\"")
```

Comment:
We should also verify that `cached` is not added by default.

```suggestion
      expect(span[:description]).to eq("SELECT \"posts\".* FROM \"posts\"")
      expect(span[:tags].key?(:cached)).to eq(false)
```